index.js

```diff
@@ -0,0 +1,56 @@
+import displayBook from './modules/displayBook.js';
+import { DateTime } from './modules/luxon.js';
+import listView from './modules/ListView.js';
+
+const bookName = document.querySelector('.bookName');
+const bookAuthor = document.querySelector('.bookAuthor');
+const form = document.querySelector('.addBook');
+const bookList = JSON.parse(localStorage.getItem('bookList')) || [];
+const message = document.querySelector('.message');
+const time = document.querySelector('.clock');
+
+class Book {
+  constructor(name, author) {
+    this.name = name;
+    this.author = author;
+  }
+
+  addBook() {
+    bookList.push(this);
+    localStorage.setItem('bookList', JSON.stringify(bookList));
+  }
+}
+
+function redirect() {
+  listView.changeContent('List');
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  message.textContent = '';
+  const book = new Book(bookName.value, bookAuthor.value);
+  book.addBook();
+  displayBook(book);
+  form.reset();
+  redirect();
+});
```

Comment:
- [ ] it would be better if you change the method and the function to `arrow function` in order to use es6 and also match the requirements